style.css

```diff
@@ -0,0 +1,784 @@
+}
+
+.footerLower img,
+.footerLower1 img {
+  width: 200px;
+}
+
+.footerUpper img,
+.footerUpper1 img {
+  width: 60px;
+}
+
+.Logo img {
+  width: 300px;
+  border: 1px solid #d3d3d3;
+  margin: 20px auto;
+}
+
+.past-events {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin-top: 15px;
+  text-align: center;
+}
+
+.past-events hr {
+  align-self: center;
+  width: 40px;
+  color: #ec5242;
+  margin: 15px 0;
+}
+
+.past-events div img {
+  width: 90%;
+  margin: 10px 0 5px 20px;
+  height: 250px;
+  opacity: 0.5;
+  transition: all 0.5s ease-in-out;
+}
+
+.footerUpper ul li img,
+.footerUpper1 ul li img {
+  width: 90px;
+  height: 90px;
+}
+
+.past-events div img:hover {
+  width: 90%;
+  margin: 10px 0 5px 20px;
+  height: 250px;
+  opacity: 1;
+  border: #36454f 1px solid;
+}
+
+.about {
+  display: flex;
+  flex-direction: column;
+}
+
+.Logo {
+  width: 90%;
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  margin: 0 auto;
+}
+
+.Logo h2 {
+  font-size: 30px;
+  margin: 30px 0 10px 0;
+}
+
+.Logo hr {
+  width: 40px;
+  color: #ec5242;
+  align-self: center;
+  margin: 5px 0 10px 0;
+}
+
+.about h1 {
+  font-size: 37px;
+  color: #ec5242;
+  font-weight: 900;
+  padding: 15px 0;
+}
+
+.about-para {
+  background: white;
+  border: #36454f;
+  padding: 5px;
+  margin-top: 10px;
+}
+
+.about-para1 {
+  padding-top: 20px;
+  font-size: 25px;
+  color: #ec5242;
+}
+
+.about-para2 {
+  margin: 20px 0;
+  width: 90%;
+  align-items: center;
+  font-size: medium;
+}
+
+.introContainer h2 {
+  padding: 15px 0;
+  font-size: 23px;
+  color: #ec5242;
+}
+
+.introContainer h1 {
+  font-size: 37px;
+  color: #ec5242;
+  font-weight: 900;
+  padding: 15px 0;
+}
+
+.para3 {
+  width: 95%;
+  font-size: medium;
+  line-height: 1.5;
+  border: #f4f4f4 3px solid;
+  margin: 10px 0;
+  background-color: white;
+  padding: 5px;
+  text-align: center;
+}
+
+.main {
+  background-image: url("./Images/diamond.png");
+  background-repeat: repeat;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: #272a31;
+}
+
+.main h1 {
+  margin-top: 40px;
+  text-align: center;
+}
+
+.join_program_link {
+  background-color: #ec5242;
+  color: #d3d3d3;
+  width: 80%;
+  display: flex;
+  height: 80px;
+  margin: 40px auto;
+  padding-left: 50px;
+  align-items: center;
+}
+
+.hidden {
+  display: none;
+}
+
+.speaker-content {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.speaker-content h2 {
+  margin-top: 40px;
+  text-align: center;
+  font-size: 35px;
+}
+
+.speaker-content hr {
+  margin: 20px 0;
+  width: 40px;
+  color: #ec5242;
+  align-self: center;
+}
+
+.speakerList {
+  display: flex !important;
+  padding: 10px;
+}
+
+.speakerList hr {
+  align-self: start;
+  color: #36454f;
+}
+
+.para {
+  color: #ec5242;
+}
+
+.footerUpper,
+.footerUpper1 {
+  background-color: #272a31;
+}
```

Comment:
- [ ] Kindly fix the `text` and `hr` tag` color for the `Partners` section on the `about` page in mobile version to match the given design.
  - Your design:
  - ![Screenshot from 2023-02-09 22-00-34](https://user-images.githubusercontent.com/60083437/217867580-95613895-e8d8-4062-9710-69aa0b97b8a2.png)
  - Given design
  - ![Screenshot from 2023-02-09 22-03-09](https://user-images.githubusercontent.com/60083437/217867965-936c09c8-44f9-425b-b34d-20d9068ab2d8.png)

Reply:
I have seen the mistake I did and I will correct that.
Thank you.